### ISSUE TITLE
Abbreviation "OAPIF" offers potential for confusion

### ISSUE THREAD

**Issue by jsaligoe** [INSPIRE-MIF/gp-ogc-api-features#24]
The need for a shortened name for “OGC API Features” is apparent, and in the draft document “OAPIF” is used. However, this abbreviation is already used as the short name for the OGC API Features GDAL Driver https://gdal.org/drivers/vector/oapif.html and therefore could be confusing from a technical perspective to call it the same thing. Note also the discussion on naming convention provided in the [implementation standard Abstract](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#abstract) "That is, the title of this document in OGC is "OGC API - Features - Part 1:Core" and the title in ISO is "Geographic Information - Geospatial API for Features - Part 1: Core.""

For consideration, use a different abbreviation, such as "OGCAPI-F" or "GeoAPI-F".

**Comment by alexanderkotsev** [comment]
Indeed, the name (OAPIF) that we use is not ideal. We would like to rely on the OGC for a shorter name and not introduce a new one ourselves. Proposals for OGC API names are already discussed in https://github.com/opengeospatial/oapi_common/issues/35. Once the short names are officially confirmed by the OGC, we can go ahead and modify our document accordingly.

**Comment by heidivanparys** [comment]
Still no resolution at OGC on the short names as far as I know. What we could do is spell the name out in the specification, or use the OGC document numbers as a reference (OGC 17-069r4, OGC 18-058, etc.). The use of OAPIF in the URIs is more tricky though.
